lab-kyle/test/main-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var main = require(__dirname + '/../main');
+var equal = require('assert').equal;
+
+describe('testing module main', function(){
+  describe('testing main function', function(){
+    it('with process.argv[2] = kyle should return hello kyle', function(){
+      const result = main();
```

Comment:
Set process.argv[2] artificially to test it in your program:

var original = process.argv[2];
process.argv[2] = kyle;
testing... (equals assertion, etc.)
process.argv[2] = original;

Make sure to reset process.argv[2] to what it was originally, this isn't necessary here but is something you should always do because other tests may involve it and it is likely you won't want them to use your modified value.